### ISSUE TITLE
LungMAP missing DRS URLs in some cases

### ISSUE THREAD

**Issue by bvizzier-ucsc** [DataBiosphere/azul#6528]
Submitted via email by Josh Fortriede
# TLDR;
- LM4 to TDR contained DRS_URIs for FASTQ files.
- File Manifest from Omics Data Browser does not have the DRS_URIs. 
- Datasets that should have BDCat DRS URIs in TDR & Data Browser
   - Human distal airways contain a multipotent secretory cell that can regenerate alveoli : f899709c-ae2c-4bb9-88f0-131142e6c7ec
   - Single-cell multiomic profiling of human lungs reveals cell-type-specific and age-dynamic control of SARS-CoV2 host genes : 20037472-ea1d-4ddb-9cd3-56a11a6f0f76

# Preface:

I did a look at TDR snapshots and the information available in the [data browser](https://data-browser.lungmap.net/projects) related to the DRS URIs generated by BioData Catalyst (BDCat) and then submitted in the file descriptor files to TDR. Below are my findings.

# Dataset:
The following datasets have been submitted to BDCat and gotten DRS URI's that we have sent to TDR
- Human distal airways contain a multipotent secretory cell that can regenerate alveoli
  - f899709c-ae2c-4bb9-88f0-131142e6c7ec
  - [Ingest Broker](https://ingest.lungmap.net/?action=projectDetails&schema=project&uuid=f899709c-ae2c-4bb9-88f0-131142e6c7ec)
  - [Data Browser](https://data-browser.lungmap.net/projects/f899709c-ae2c-4bb9-88f0-131142e6c7ec)
  - [TDR Snapshot](https://data.terra.bio/snapshots/15324e53-b8d3-40b7-92fb-abfeb35e6515/data)
- Single-cell multiomic profiling of human lungs reveals cell-type-specific and age-dynamic control of SARS-CoV2 host genes
  - 20037472-ea1d-4ddb-9cd3-56a11a6f0f76
  - [Ingest Broker](https://ingest.lungmap.net/?action=projectDetails&schema=project&uuid=20037472-ea1d-4ddb-9cd3-56a11a6f0f76)
  - [Data Browser](https://data-browser.lungmap.net/projects/20037472-ea1d-4ddb-9cd3-56a11a6f0f76)
  - [TDR Snapshot](https://data.terra.bio/snapshots/a35e4770-1131-4dc3-955d-f0e7f526dc21/data)

## TDR Snapshot [LM4](https://data.terra.bio/snapshots/15324e53-b8d3-40b7-92fb-abfeb35e6515/data):

When Viewing the "sequence_file", the descriptor column shows the JSON content of the file descriptor. This shows the DRS URI.
 
![image](https://github.com/user-attachments/assets/10651676-ebc7-4260-a270-964254e76c75)

## File Manifest from Respiratory Airway Secretory Cells Submission - [OMIC Data Browser](https://data-browser.lungmap.net/projects/f899709c-ae2c-4bb9-88f0-131142e6c7ec/project-metadata)

The DRS URI is missing.
![image](https://github.com/user-attachments/assets/312581d9-ac6d-4e71-8493-f96c763a0c55)

**Comment by hannes-ucsc** [comment]
This is essentially a duplicate of https://github.com/DataBiosphere/azul/issues/3631 which is assigned to @bvizzier-ucsc (the author of this issue) in order to [obtain](https://github.com/DataBiosphere/azul/issues/3631#issuecomment-1894456815) an answer to my [question](https://github.com/DataBiosphere/azul/issues/3631#issuecomment-1888288320) regarding the expected level of support for these types of DRS URIs.

**Comment by dsotirho-ucsc** [comment]
Closing as dup of #3631
